src/screens/Settings/components/WineSettings/index.tsx

```diff
@@ -74,21 +74,23 @@ export default function WineSettings({
   function removeCustomPath() {
     const newPaths = customWinePaths.filter((path) => path !== selectedPath)
     setCustomWinePaths(newPaths)
-    return setSelectedPath(customWinePaths.length ? customWinePaths[0] : '')
+    return setSelectedPath(newPaths.length ? newPaths[0] : '')
```

Comment:
why the change?

Reply:
Year now i see this can lead to problems. Will revert that and adapt the test.

Reply:
ok

Reply:
Reverted.